src/main/java/mail/GeneratorPassword.java

```diff
@@ -0,0 +1,22 @@
+package mail;
+
+import java.util.Random;
+
+public class GeneratorPassword {
+
+    public String generatedPassword() {
+
+        String upperWords = "QWERTYUIOPASDFGHJKL";
+        String lower = "qwertyuiopasdfghjkl";
```

Comment:
1. Use the same style everywhere:
`upperWords` and `lowert`

2. Can we use one string for lower and uppercase?

Reply:
3. Think about variables. Maybe we can move something to class level